test/jdk/jdk/internal/platform/docker/TestDockerMemoryMetrics.java

```diff
@@ -102,6 +102,21 @@ private static void testMemoryLimit(String value) throws Exception {
 
     private static void testMemoryFailCount(String value) throws Exception {
         Common.logNewTestCase("testMemoryFailCount" + value);
+
+        // Check whether swapping really works for this test
```

Comment:
Please explain what "swapping not working" actually means in this comment. One version of it is already handled via JDK-8250984. Suggestion: "On some systems there is no swap space enabled. On those systems running `java -version`??? with a memory limit fails due to swap space size being 0". Or something like that.

Reply:
> Please explain what "swapping not working" actually means in this comment.

Updated.
Thanks.